domain/src/main/java/com/jakefoundation/buildabotworkshop/domain/Bot.java

```diff
@@ -0,0 +1,13 @@
+package com.jakefoundation.buildabotworkshop.domain;
+
+public interface Bot {
+    {
+        public void moveCommand(GameState);
+        public void fireCommand(GameState);
+        public void String getBotCommonName;
+        public void String getOwnerName;
+        public void String getOwnerPassword;
```

Comment:
These return String, not void. You can't have fields on an interface, so these are getter methods with no parameters.